ibtk/include/ibtk/HierarchyMathOps.h

```diff
@@ -1492,6 +1489,12 @@ class HierarchyMathOps : public SAMRAI::tbox::DescribedClass
     // Housekeeping.
     std::string d_object_name;
 
+    // Destination fill box ghost cell width.
+    const int d_dst_fill_box_gcw;
```

Comment:
```suggestion
    int d_dst_fill_box_gcw;
```
I know we don't permit copying these things but making them `const` is still weird.